packages/collector/src/announceCycle/agentready.js

```diff
@@ -146,9 +146,15 @@ function sendTracingMetrics() {
   const payload = tracing._getAndResetTracingMetrics();
   agentConnection.sendTracingMetricsToAgent(payload, error => {
     if (error) {
-      logger.warn('Error received while trying to send tracing metrics to agent: %s', error.message);
+      logger.info(
```

Comment:
I downgraded this from warning to info. Tracemetrics are for our internal self-monitoring and not a direct concern for users of `@instana/collector`.